raw-svo.setup.lua

```diff
@@ -188,11 +188,43 @@ end)
 signals.gmcproominfo        = luanotify.signal.new()
 signals.gmcpcharstatus      = luanotify.signal.new()
 signals.gmcpcharitemslist   = luanotify.signal.new()
+signals.gmcpcharitemslist:connect(function()
+  if not gmcp.Char.Items.List.location then echof("(GMCP problem) location field is missing from Achaea's response.") return end
```

Comment:
This has the potential to spam the user with a lot of error messages they wouldn't understand (and in this case, able to deal with), not really part of the current philosophy. Best to use a debugf here, so if something is wrong, you can look in the debug output and figure it out.